lib/ecstatic/show-dir/index.js

```diff
@@ -24,6 +24,15 @@ module.exports = function (opts, stat) {
 
   return function middleware (req, res, next) {
 
+    if (req.url.endsWith('?css')) {
+      res.writeHead(200, {
+        'Content-Type': 'text/css',
+        'Cache-Control': 'public, max-age=31536000'
```

Comment:
I though because the styles are built into the server and are not going to change anytime soon, it makes sense to cache them for a long time.